data/npc/lib/npcsystem/keywordhandler.lua

```diff
@@ -44,6 +44,10 @@ if not KeywordHandler then
 	end
 
 	-- Returns the parent of this node or nil if no such node exists.
+	function KeywordNode:hasParent()
+		return self.parent
```

Comment:
This should return a booean, e.g. `return self.parent ~= nil`

Reply:
Ready!